kubernetes/loculus/values.schema.json

```diff
@@ -245,14 +245,72 @@
                   "type": "boolean",
                   "default": true,
                   "description": "If false, the submission form will not allow submission of consensus sequences (i.e. the sequences file must be omitted). All consensus sequence related parts on the website will be hidden."
+                },
+                "files": {
+                  "type": "object",
+                  "properties": {
+                    "enabled": {
+                      "groups": ["schema"],
+                      "type": "boolean",
+                      "default": false,
+                      "description": "If true, enable support for submitting additional fields. S3 needs to be configured and fields need to be configured under `fields`."
+                    },
+                    "fields": {
+                      "groups": ["schema"],
+                      "type": "array",
+                      "description": "The file fields that can be submitted for this organism.",
+                      "items": {
+                        "type": "object",
+                        "placeholder": "fileField",
+                        "properties": {
+                          "name": {
+                            "groups": ["schema"],
+                            "type": "string",
+                            "description": "The name of the file field, i.e. 'raw_reads'."
+                          },
+                          "multipleFiles": {
+                            "groups": ["schema"],
+                            "type": "boolean",
+                            "description": "Whether this field supports submission of multiple files."
+                          }
+                        },
+                        "additionalProperties": false
+                      }
+                    }
+                  },
+                  "additionalProperties": false
                 }
-              }
+              },
+              "additionalProperties": false
             },
             "description": {
               "groups": ["schema"],
               "docsIncludePrefix": false,
               "type": "string"
             },
+            "files": {
+              "groups": ["schema"],
+              "docsIncludePrefix": false,
+              "description": "An array of output files.",
+              "type": "array",
+              "placeholder": "file",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "groups": ["schema"],
+                    "type": "string",
+                    "description": "TODO"
+                  },
+                  "multipleFiles": {
+                    "groups": ["schema"],
+                    "type": "boolean",
+                    "description": "TODO"
+                  }
+                },
+                "additionalProperties": false
+              }
+            },
```

Comment:
This looks like a duplication of the above files section, we should avoid having duplicates as they make our already verbose config even bigger - also things could go bad if one accidentally only changes one not the other of the duplicate parts.

Reply:
I think what you are looking at above is the `submissionDataTypes`'s `files` section? If so, this is completely distinct: you could disable submitting files but enable outputting files, for example allowing the pipeline to take consensus sequences and generate e.g. PNGs with graphs about the consensus sequences, or you could enable inputting files but disable outputting files, i.e. use the raw reads to create a consensus sequence but not make them available for download.

Reply:
I've added some admin docs for the config, that also mention this: https://github.com/loculus-project/loculus/pull/3978/commits/cb2f35c115509d01dca1ea472615e6e13eec9139